app/public/css/styles.css

```diff
@@ -1477,3 +1476,65 @@ p {
   background-position: center;
   background-image: url('../assets/input_pencil_icon.svg');
 }
+
+/* CARD */
+
+.card {
+  background-color: white;
+  padding: 40px 56px;
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.08);
+}
+
+/* TRANSACTIONS */
+
+.transactions {
+  padding: 40px 56px;
+}
+
+.transactions-title {
```

Comment:
This is the same for all `.card`s.

Please, rename it to `.card-title` and move it to CARD.